src/containers/PlayBackPage/Comment/actions.js

```diff
@@ -37,8 +38,47 @@ function setCurrentRecordWithHistory(result) {
   };
 }
 
+function setSnapComments(snapComments) {
+  return {
+    type: 'SET_SNAP_COMMENTS',
+    snapComments,
+  };
+}
+export function setCategoryIndex(index) {
+  return {
+    type: 'SET_CATEGORY_INDEX',
```

Comment:
Extract `SET_CATEGORY_INDEX` to constants.js, then you can use in `actions.js` and `reducer.js` file